test/complexity_test.cc

```diff
@@ -52,7 +56,7 @@ int AddComplexityTest(std::string test_name, std::string big_o_test_name,
 // --------------------------- Testing BigO O(1) --------------------------- //
 // ========================================================================= //
 
-void BM_Complexity_O1(benchmark::State& state) {
+void BM_Complexity_O1(benchmark::State &state) {
```

Comment:
why did these &s change? we should be normalised across the project to & with the type, afaik.

Reply:
That's what the clang format did, but I'll change it back!

Reply:
clang format will change things to the perceived "majority" of the file. i guess this file is mostly wrong :P